python/pip_install/pip_repository.bzl

```diff
@@ -377,8 +381,8 @@ def _pip_repository_bzlmod_impl(rctx):
             "@{}//{}:whl".format(repo_name, p) if rctx.attr.incompatible_generate_aliases else "@{}_{}//:whl".format(rctx.attr.name, p)
             for p in bzl_packages
         ]),
-        "%%NAME%%": rctx.attr.name,
         "%%REQUIREMENTS_LOCK%%": str(requirements_txt),
+        "%%TMPL%%": macro_tmpl,
```

Comment:
Lets use a more descriptive name than "TMPL" as the substituted name. A template that templates in a a variable named template is a bit ~~meta-templately~~ confusing :).